### ISSUE TITLE
FineTunning the DetectNet

### ISSUE THREAD

**Issue by VanitarNordic** [NVIDIA/DIGITS#1383]
Hello,

I read the tutorial about DetecNet and KITTI images. a question pops up.

if the intention is to fine-tune or transfer learn the Detectnet, then why you have not renamed some layers?!

**Comment by gheinrich** [comment]
Can you more specific as to which layers you think we should have renamed?

**Comment by VanitarNordic** [comment]
a pre-trained network is trained on a different Dataset, therefore all layers may have not suitable for a new dataset.

I can tell you about the GoogLeNet classification, the layers `loss1/classifier`, `loss2/classifier`, and `loss3/classifier` must be renamed, but in a case that the new dataset objects are almost the same with pretrained ones. then the trained weight would be useful. Then the network can be retrained with low number of images also with a high accuracy.

Therefore there should be some layers in the DetectNet prototxt file that gonna be renamed. I assume there should be final layers.

**Comment by gheinrich** [comment]
Hello, Detectnet reuses the feature extractor part of GoogLeNet i.e. the inception modules up to layer `inception_5b/output`. Layers from the GoogLeNet classifier do not have to be renamed as they are simply not present in the definition of Detectnet.

You are welcome to make changes to Detectnet to improve its accuracy, in which case we'd be glad to receive a contribution from you. In the meantime, it feels like this issue can be closed?

**Comment by VanitarNordic** [comment]
Yes, I mentioned that my description is about GoogLeNet and classifier, but I had asked you about DetectNet. Thank you for the reply.

Anyway, one question remains, then you can close this. 

How much accuracy dos the DetectNet achieved on famous Datasets such as VOC and similar or in competitions?

**Comment by gheinrich** [comment]
> How much accuracy does the DetectNet achieved on famous Datasets such as VOC and similar or in competitions?

I do not know but you can ask on the DIGITS users list whether somebody tried this:
https://groups.google.com/forum/#!forum/digits-users
